internal/services/stock_exchange/stock_exchange.go

```diff
@@ -1,5 +1,22 @@
 package stock_exchange
 
+import (
+	"github.com/jinzhu/gorm"
+	"log"
+)
+
+var DB *gorm.DB
```

Comment:
Why is this object defined here? This already has been initialised in config file.

Reply:
Why this file is present in your PR?